projects/js-packages/publicize-components/src/components/form/broken-connections-notice.tsx

```diff
@@ -73,17 +73,18 @@ export const BrokenConnectionsNotice: React.FC = () => {
 				{ __( 'Your following connections are broken:', 'jetpack' ) }
 				<ul>
 					{ Object.entries( brokenConnectionsList ).map( ( [ service_name, connectionsList ] ) => {
-						const service = servicesMap[ service_name ];
-
-						if ( ! service ) {
-							return null;
-						}
+						const serviceLabel =
+							// For Jetpack sites, we should have the service in the map
+							// But for WPCOM sites, we might not have the service in the map yet
+							servicesMap[ service_name ]?.label ||
+							// So we capitalize the service name
+							service_name[ 0 ].toUpperCase() + service_name.substring( 1 );
 
 						return (
-							<li key={ service.ID }>
+							<li key={ service_name }>
```

Comment:
Can this throw a warning in the presence of duplicate services?

Reply:
If you notice above, it's actually the object key and that's unique.